modules/component-store/schematics-core/utility/project.ts

```diff
@@ -30,8 +30,8 @@ export function getProjectPath(
 ) {
   const project = getProject(host, options);
 
-  if (project.root.substr(-1) === '/') {
-    project.root = project.root.substr(0, project.root.length - 1);
+  if (project.root.substring(-1) === '/') {
```

Comment:
`substring` will not work with negative index.

```suggestion
  if (project.root.slice(-1) === '/') {
```

 The same should be applied to other places where `string.substr(-1)` is used.